e2eTest/project_test.go

```diff
@@ -202,3 +202,46 @@ func TestProjects(t *testing.T) {
 	})
 
 }
+
+func TestGetProjectList(t *testing.T) {
```

Comment:
It would be good idea to add a test where two users create one project and then to validate that get project list truly only returns the project for the specified user not all of them.

Reply:
Added